_posts/2025-04-04-smm9509-krita-453572.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+title:  "Improving Translatablility and Usability in the JPEG-XL Export Menu in Krita"
+
+categories: 
+- Contribution
+
+# Enter your name below
+author: Elizabeth Myakisheva-Rempel
+---
+
+For my contribution, I picked Krita because it's an app I use a lot for handwriting and illustrations on my engineering homework. I considered implementing a feature I want (PDF import and export useful for note-taking), but the maintainers intentionally dropped support for pdf export in the past. 
+
+Krita passed the Commarch considerations with flying colors. The only points of concern was the very large amount of source code and the primary branch's name being "master". It was also well recommended by one of the commarch presentations we did in class. Krita's community type is Federation. 
+
+## Finding the bug to fix
+
+After that consideration, I searched and browsed the [bug tracker](https://bugs.kde.org/).
+
+![searching for krita issues in the bug tracker](.attachments.4761187/image%20%282%29.png)
+
+After browsing the bugs, I found one I liked.
+
+![image (3).png](.attachments.4761187/image%20%283%29.png)
+
+I picked bug [453572](https://bugs.kde.org/show_bug.cgi?id=453572), that I felt would be easy and valuable to solve.
+
+The bug 453572, titled "JPEG-XL advanced export options confusing" complains about vocabulary being too technical so that it's difficult to understand and localize.
+
+![screenshot of bug report](.attachments.4761187/image.png)
```

Comment:
These images dont appear to be showing up on my end - did you commit them to your branch?